### ISSUE TITLE
[[MAS2.4.7][Keyboard Navigation- Open Bot Dialog] Focus is not visible on "Close" button.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1951]
**Actual Result:**
When we press "Shift + Tab" after the "Bot URL" edit field, the focus should visible on the "X" i.e. close button but the focus is not visible their where as if user press "Enter" the pop-up/dialog gets closed.

**Expected Result:**
After "Bot URL" when user press "Shift + Tab", the focus should lands on the "X" button as well as the focus should be clearly visible on the "X" button.

**Note:**
Same issue observed with each pop-up and dialog, throughout the application.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the Welcome tab and reach at "Open Bot" button and activate it.
4. "Open a Bot" dialog gets appeared, navigate through the dialog.
5. Observe the issue.

**User Impact:**
When a component can receive keyboard focus and the focus is not visible to the user. User will not able to activate the component using the keyboard. 

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment:**
[Focus is not visible on Close button.zip](https://github.com/microsoft/BotFramework-Emulator/files/3789397/Focus.is.not.visible.on.Close.button.zip)

**Comment by tonyanziano** [comment]
This seems to be fixed in the latest nightly and is ready to be verified.

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0.94518
